winforms/src/toga_winforms/app.py

```diff
@@ -338,6 +361,21 @@ def hide_cursor(self):
             WinForms.Cursor.Hide()
         self._cursor_visible = False
 
+    def winforms_DisplaySettingsChanged(self, sender, event):
+        for window in self.interface.windows:
+            window._impl.update_scale(
+                screen=WinForms.Screen.FromControl(window._impl.native)
```

Comment:
This calls `update_scale` on the Window, but how is that method called on the other Scalable subclasses like Widget and Dialog?

Reply:
The Widgets uses the DPI scale of window on which they are present.